catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -1133,6 +1173,12 @@ private FilterCapabilities buildFilterCapabilities() {
    */
   private OperationsMetadata buildOperationsMetadata() {
 
+    List<String> typeNames =
+        Stream.of(getQueryFilterTransformerTypeNames(), TYPE_NAMES_LIST)
+            .flatMap(Collection::stream)
+            .distinct()
```

Comment:
Might want to consider using a set instead

Reply:
done